mgr/ConsDefs.pm

```diff
@@ -801,6 +801,7 @@
 	# restore  
 	($CFLAGS,$CXXFLAGS) = @ARGS;        # <--- same order than above
     }
+ $CXXFLAGS .= " -fdiagnostics-color=always";#  -Wno-pedantic -Wno-shadow -Wno-unused-parameter -Wno-extra -Wno-shadow";
```

Comment:
Compilation error on SDCC (I'm trying to build an SL24y test library with this branch):
```
g++ -m32 -fPIC -pipe -Wall -Woverloaded-virtual -std=c++0x -Wno-long-long -pthread -Wno-deprecated-declarations -fdiagnostics-color=always-DEVTGEN_EXTERNAL=1 -DEVTGEN_PYTHIA=1 -DEVTGEN_PHOTOS=1 -DEVTGEN_TAUOLA=1 -DEvtGen_version=\"EvtGen1_06_00\" -DPythia8_version=\"Pythia8_1_86\" -g -Dsl73_gcc485 -D__ROOT__ -DNEW_DAQ_READER -I. -IStRoot -I.sl73_gcc485/include -IStRoot/StarGenerator/EvtGen1_06_00 -IStRoot/StarGenerator/HepMC2_06_09 -IStRoot/StarGenerator/Pythia8_1_86/include -IStRoot/StarGenerator/Photos3_61/include -IStRoot/StarGenerator/Tauola1_1_5/include -I/afs/rhic.bnl.gov/star/packages/SL24y -I/afs/rhic.bnl.gov/star/packages/SL24y/StRoot -I/afs/rhic.bnl.gov/star/packages/SL24y/.sl73_gcc485/include -I/cvmfs/star.sdcc.bnl.gov/star-spack/spack/opt/spack/linux-rhel7-x86/gcc-4.8.5/root-5.34.38-llsepmmfuwlsucogcwbjiodncxanoudt/include -I/afs/rhic.bnl.gov/star/packages/SL24y/StRoot/StarGenerator/EvtGen1_06_00 -I/afs/rhic.bnl.gov/star/packages/SL24y/StRoot/StarGenerator/HepMC2_06_09 -I/afs/rhic.bnl.gov/star/packages/SL24y/StRoot/StarGenerator/Pythia8_1_86/include -I/afs/rhic.bnl.gov/star/packages/SL24y/StRoot/StarGenerator/Photos3_61/include -I/afs/rhic.bnl.gov/star/packages/SL24y/StRoot/StarGenerator/Tauola1_1_5/include -c .sl73_gcc485/obj/StRoot/StarGenerator/EvtGen1_06_00/StarEvtGenDecayer.cxx -o .sl73_gcc485/obj/StRoot/StarGenerator/EvtGen1_06_00/StarEvtGenDecayer.o
g++: error: unrecognized argument in option '-fdiagnostics-color=always-DEVTGEN_EXTERNAL=1'
g++: note: valid arguments to '-fdiagnostics-color=' are: always auto never
cons: *** [.sl73_gcc485/obj/StRoot/StarGenerator/EvtGen1_06_00/StarEvtGenDecayer.o] Error 1
cons: errors constructing .sl73_gcc485/obj/StRoot/StarGenerator/EvtGen1_06_00/StarEvtGenDecayer.o
```

Reply:
The problem is missing space between "-fdiagnostics-color=always" and "-DEVTGEN_EXTERNAL=1' coming from mgr/Conscript-standard:608:        $CXXFLAGS .= "-DEVTGEN_EXTERNAL=1 -DEVTGEN_PYTHIA=1 -DEVTGEN_PHOTOS=1 -DEVTGEN_TAUOLA=1 -DEvtGen_version=\\\"$pkg\\\" ";

Reply:
I have committed the " " before -DEVTGEN_EXTERNAL=1 -...